src/sidebars/version-directive.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.version')
+        .value('version', '0.1.0')
```

Comment:
@WPFerg I strongly believe that we should match this number to our GH number, not sub it artificially. I don't see a problem with a first release being version 11.0.0 or something like this. Otherwise this number doesn't help much.